docs/synapse/userguides/storm_ref_lift.rstorm

```diff
@@ -356,6 +499,55 @@ Lift by Secondary Property Value
   the wildcard expression. See the type-specific documentation for :ref:`type-time` types for a detailed discussion of
   these behaviors.
 
+.. _lift-prop-std-interface:
+
+Lift by Interface Property Value
+++++++++++++++++++++++++++++++++
+
+If a form inherits an :ref:`interface`, you can lift all nodes of all forms with a specific interface-derived property value
+by using the name of the interface.
+
+.. TIP::
+  
+  You must specify the interface where the property is defined. If an interface inherits another interface, you cannot
+  perform this type of lift using an interface and an inherited property name.
+  
+  Synapse returns results in lexical order (sorted, ascending to descending) based on the way the queried property is
+  indexed. When using an interface to lift by secondary property, the full set of results is combined before being
```

Comment:
The full set of results isn't combined+sorted+returned, since the values are indexed in lexical order, it is iterating them all in parallel and yielding the results in order.

Reply:
Attempted to clarify - not sure if I splained / captured it. :-)

Reply:
How about this, tweaked it a little bit to use "lifts":
"When using an interface to lift by secondary property, Synapse performs the lifts for each form in parallel, and yields the results in order."